packaging/greenboot/microshift-running-check.sh

```diff
@@ -10,9 +10,18 @@ PODS_CT_LIST=(2                        1                    1                 2
 # shellcheck source=packaging/greenboot/functions.sh
 source /usr/share/microshift/functions/greenboot.sh
 
+# Set the term handler to convert exit code to 1
+trap 'return_failure' TERM
+
 # Set the exit handler to log the exit status
 trap 'script_exit' EXIT
```

Comment:
Is `script_exit` called when TERM arrives?

Reply:
Yes, here's an example:
```
[root@microshift-dev microshift]# /etc/greenboot/check/required.d/40_microshift_running_check.sh
STARTED
GRUB boot variables:
boot_success=1
boot_indeterminate=0
Greenboot variables:
None
error: loading sysroot: opendir(ostree/repo): No such file or directory
The ostree status:

Waiting 300s for MicroShift service to be active and not failed
The microshift.service systemd unit is failed. Terminating...
FAILURE
[root@microshift-dev microshift]# echo $?
1
```